keyboards/gray_studio/bd60/info.json

```
@@ -0,0 +1,552 @@
+{
+    "keyboard_name": "BD60",
+    "url": "",
+    "maintainer": "edwardslau",
+    "manufacturer":"GrayStudio",
+    "processor": "STM32F072",
+    "diode_direction": "COL2ROW",
+    "bootloader": "stm32-dfu",
+       "usb": {
+        "vid":"0x4753",
+        "pid":"0x4160",
+        "device_version": "0.0.1"
+    },
+    "features": {
+        "backlight": false,
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true,
+        "rgblight": true
+    },
+    "ws2812": {
+        "pin": "B10"
+    },
+    "matrix_pins": {
+        "cols": ["A3", "C14", "C13", "B9", "B8", "B7", "B6", "B5", "B4", "B3", "A15", "B0", "A8", "B15", "B14"],
+        "rows": ["B13", "B12", "A2", "A0", "A1"]
+    },
+"rgblight": {
+        "led_count": 1,
+        "sleep": true,
+        "max_brightness": 150,
+        "layers": {
+            "enabled": true,
+            "override_rgb": true
+        },
+        "animations": {
+            "alternating": true,
+            "breathing": true,
+            "christmas": true,
+            "knight": true,
+            "rainbow_mood": true,
+            "rainbow_swirl": true,
+            "rgb_test": true,
+            "snake": true,
+            "static_gradient": true,
+            "twinkle": true
+        }
+    },
```

Im guessing `LOCKING_SUPPORT_ENABLE` is more copy paste than intentional, but...

```suggestion
    },
    "qmk": {
        "locking": {
            "enabled": true,
            "resync": true
        }
    },
```